src/test/resources/Features/OCD-2134-batchWithdraw.feature

```diff
@@ -1,8 +1,24 @@
-Feature: Batch update Certification Status to 'Withdrawn by Developer' for a set of Cerner and Intermountain Healthcare listings
+Feature: OCD-2134 - Batch update of Certification Status to 'Withdrawn by Developer'
+		Batch update Certification Status to 'Withdrawn by Developer' for a set of Cerner and Intermountain Healthcare listings
+		
+Background: User is on search page and 'Withdrawn by Developer' filter option is checked
+	Given I am on CHPL search page
+	And I look at Certification Status options
+	And I apply "Withdrawn_by_Developer" filter
```

Comment:
The Features files should read as if they were written by a human, and given to a non-programmer. To a non-programmer, the filter should be `Withdrawn By Developer` without the underscores.